README.md

```diff
@@ -96,17 +137,18 @@ To create a new organization or import an existing organization, add the followi
 
 ```typescript
 const organization = new Organization(stack, "Organization", {
-  featureSet: FeatureSet.ALL,
+  featureSet: FeatureSet.ALL, // (default) required later on to enable SCPs, enable AWS services or deletage an adminsitrator account
 });
+organization.root; // The organization's root is automatically created
 ```
 
-- `FeatureSet.ALL` is required for advanced features like Service Control Policies and is the [preferred way to work with AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_support-all-features.html)
-- The account which deploys the stack automatically becomes the management account of the new organization.
-- If an organization already exists, it will be automatically imported. The account which deploys the stacks must be the management account.
-- If the construct gets removed from the stack the organization still remains and must be manually deleted.
-- For deletion of an organization you must previously remove all the member accounts, OUs, and policies from the organization.
-- Currently, you can have only one root. AWS Organizations automatically creates it for you when you create the new organization.
-- It can only be used from within the management account in the us-east-1 region.
+- `FeatureSet.ALL` is required for advanced features like Service Control Policies (SCP) and is the [preferred way to work with AWS Organizations](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_support-all-features.html)
+- The account which deploys the stack, will become automatically the management account of the new organization.
+- If an organization already exists, it will be automatically imported. You may disable this behaviour by passing `importOnDuplicate: false` in the props.
+- If the construct gets removed from the stack the organization still remains and must be manually deleted. For deletion of an organization you must previously remove all the member accounts, OUs, and policies from the organization. [Deleting the organization by removing the management account](https://docs.aws.amazon.com/organizations/latest/userguide/orgs_manage_org_delete.html)
```

Comment:
If the construct is removed from the stack the organization will remain and must be deleted manually.